core/chains/evm/txmgr/broadcaster_benchmark_test.go

```diff
@@ -9,8 +9,11 @@ import (
 
 	gethCommon "github.com/ethereum/go-ethereum/common"
 	"github.com/jmoiron/sqlx"
+
 	"github.com/smartcontractkit/chainlink-evm/pkg/keys"
 	"github.com/smartcontractkit/chainlink-evm/pkg/keys/keystest"
+	evmtxmgrtestutils "github.com/smartcontractkit/chainlink/v2/core/chains/evm/txmgr/testutils"
```

Comment:
Let's name this package something that doesn't require aliasing, like `txmgrtest`:
```suggestion
	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/txmgr/txmgrtest"
```

Reply:
Done